exotations/solvers/exotica_levenberg_marquardt_solver/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(exotica_lm_solver)
```

Comment:
Project name does not match the directory name. Perhaps change to ``exotica_levenberg_marquardt_solver`` as in the folder name (more descriptive)